media-libs/partio/partio-1.1.0.recipe

```diff
@@ -0,0 +1,107 @@
+SUMMARY="A library for particle IO and manipulation"
+DESCRIPTION="The goal of Partio is to provide a unified interface akin to \
+unified image libraries that makes it easier to load, save, and manipulate \
+particle files.
+
+Major Features:
+
+* Supports Houdini's GEO and BGEO, Maya's PDB and PDA, RenderMan's PTC
+* Arbitrary back end in-memory/cached formats can be supported from a single \
+interface
+* A Python API for easy scripting of particle manipulation
+* A C++ API for high performance particle manipulation
+* Nearest Neighbor Lookups for density estimation, Voronoi computation, etc."
+HOMEPAGE="https://www.disneyanimation.com/technology/partio.html"
+COPYRIGHT="2010-2012 Disney Enterprises"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/wdas/partio/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256='133f386f076bd6958292646b6ba0e3db6d1e37bde3b8a6d1bc4b7809d693999d'
+PATCHES="partio-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	partio$secondaryArchSuffix = $portVersion compat >= 1
+	cmd:partattr$commandSuffix = $portVersion
+	cmd:partconv$commandSuffix = $portVersion
+	cmd:partinfo$commandSuffix = $portVersion
+	cmd:partview$commandSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libGLU$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	partio${secondaryArchSuffix}_devel = $portVersion
+	devel:libpartio$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	partio$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+#	cmd:doxygen
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python
+	cmd:swig
+	"
+
+defineDebugInfoPackage partio$secondaryArchSuffix \
+	$prefix/bin/partattr$commandSuffix \
+	$prefix/bin/partconv$commandSuffix \
+	$prefix/bin/partinfo$commandSuffix \
+	$prefix/bin/partview$commandSuffix
+
+BUILD()
+{
+	mkdir -p build
+	cd build
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-Wno-dev
+
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mkdir -p $includeDir
+	mv $prefix/include/* $includeDir/
+	rm -rf $prefix/include $prefix/test
```

Comment:
I believe this would be the correct solution https://github.com/wdas/partio/pull/68